pyvista/core/utilities/geometric_sources.py

```diff
@@ -2249,6 +2275,96 @@ def j_resolution(self, j_resolution: int):
         """
         self.SetYResolution(j_resolution)
 
+    @property
+    def center(self) -> Sequence[float]:
+        """Get the center in ``[x, y, z]``.
+
+        The center of the plane is translated to the specified point.
+
+        Returns
+        -------
+        sequence[float]
+            Center in ``[x, y, z]``.
+        """
+        return self.GetCenter()
+
+    @center.setter
+    def center(self, center: Sequence[float]):
+        """Set the center in ``[x, y, z]``.
+
+        Parameters
+        ----------
+        center : sequence[float]
+            Center in ``[x, y, z]``.
+        """
+        self.SetCenter(center)
+
+    @property
+    def origin(self) -> Sequence[float]:
+        """Get the origin in ``[x, y, z]``.
+
+        Returns
+        -------
+        sequence[float]
+            Origin in ``[x, y, z]``.
+        """
+        return self.GetOrigin()
+
+    @origin.setter
+    def origin(self, origin: Sequence[float]):
+        """Set the origin in ``[x, y, z]``.
+
+        Parameters
+        ----------
+        origin : sequence[float]
+            Origin in ``[x, y, z]``.
+        """
+        self.SetOrigin(origin)
+
+    @property
+    def point_a(self) -> Sequence[float]:
+        """Get the Location in ``[x, y, z]``.
+
+        Returns
+        -------
+        sequence[float]
+            Location in ``[x, y, z]``.
+        """
+        return self.GetPoint1()
+
+    @point_a.setter
+    def point_a(self, point_a: Sequence[float]):
+        """Set the Location in ``[x, y, z]``.
+
+        Parameters
+        ----------
+        point_a : sequence[float]
+            Location in ``[x, y, z]``.
+        """
+        self.SetPoint1(*point_a)
```

Comment:
No need to unpack, the vtk method is overloaded for both `SetPoint1(float, float, float)` and `SetPoint1((float, float, float))` (I also tested this locally)
```suggestion
        self.SetPoint1(point_a)
```